### ISSUE TITLE
Suggestion: Add pyinstaller hook for css and icon files

### ISSUE THREAD

**Issue by sdobber** [niklashenning/pyqttoast#15]
When compiling `pyqttoast` with pyinstaller, the css and icon files are missing. I suggest to add a hook file to get them included in the package. I am not too much into the internals of pyinstaller - the following code in a `hook-pyqttoast.py`-file works fine for me:
```python
from PyInstaller.utils.hooks import collect_data_files
datas = collect_data_files('pyqttoast', includes=['**/*.css', '**/*.png'])
```

**Comment by niklashenning** [comment]
Thanks for letting me know, I'm looking into it.

**Comment by niklashenning** [comment]
A PyInstaller hook is now included in v1.3.1
